### PR TITLE
fix logic when EDITOR variable is unset

### LINKED ISSUE #1: EDITOR varable unset by default causing error in settitle()
the scripts here do not set EDITOR variable but assume that it is set elsewhere. however this is not the case by default at least with a clean ubuntu 22.04 install. when EDITOR variable is unset, settitle() function begins throwing error "basename: invalid option"

could either have settitle() understand empty EDITOR variable, or could assign a default when it's unset

### PR DESCRIPTION
**Pull request by ardentperf**
a straightforward way to fix #1

### CODE REVIEW

File: .homeconfig [hunk 1]
```diff
@@ -55,7 +55,7 @@ export HOME_POSTGRES_CFLAGS=""
 export HOME_POSTGRES_LDFLAGS=""
 
 # Custom options for Postgres ./configure
-export HOME_POSTGRES_CONFIGURE=""
+export HOME_POSTGRES_CONFIGURE="--enable-tap-tests"
```
Comment (michaelpq) [inline .homeconfig:58]: The answer to that is "force it in your own private profile". This code is intentional.

File: .bashrc [hunk 2]
```diff
@@ -206,3 +206,13 @@ fi
 if [ -f "$HOME_CWD" ]; then
 	cd "$(< ${HOME}/.cwd)"
 fi
+
+# Create directory if it doesn't exist
+if ! [ -d "$HOME_GIT" ]; then
+	mkdir -vp $HOME_GIT
+fi
+
+# Download to avoid mixing licenses in this repo
```
Comment (michaelpq) [inline .bashrc:215]: Downloading data from potentially untrusted sources is not in the spirit.

File: .bashrc [hunk 3]
```diff
@@ -206,3 +206,13 @@ fi
 if [ -f "$HOME_CWD" ]; then
 	cd "$(< ${HOME}/.cwd)"
 fi
+
+# Create directory if it doesn't exist
```
Comment (michaelpq) [inline .bashrc:210]: Ah, makes sense.

File: .bashrc [hunk 4]
```diff
@@ -110,7 +110,7 @@ function settitle () {
 		# and "git open" are exceptions present in this environment so
 		# handle them as well. If there is no match just use the first
 		# command launched and show it in title.
-		if [[ "${CURRENT_COMMAND}" == "${EDITOR}"* || \
+		if [[ "${CURRENT_COMMAND}" == "${EDITOR:-xxxxxxxx}"* || \
```
Comment (michaelpq) [inline .bashrc:113]: Indeed, the comparison can be improved here.

**Comment by ardentperf** [comment]
Ack. I'm very sorry... I forgot how github pull requests work.

When I created this pull request there was only a single commit.  I forgot that when I later added more commits to my branch, it would add them to the pull request.

The only thing I actually intended to have in the pull request was "${EDITOR:-xxxxxxxx}" !!

I can close this and create a new one with a dedicated branch if you want... apologies.  😔

**Comment by ardentperf** [comment]
I guess, since you looked at everything anyway, don’t have any objections if my new PR also includes the “mkdir git” and a copy of screenrc (a full copy, not this code here which downloads one)?

**Comment by ardentperf** [comment]
I guess, since you looked at everything anyway, do you have any objections if my new PR also includes the “mkdir git” and a copy of screenrc (a full copy, not this code here which downloads one)?

**Comment by michaelpq** [comment]
screenrc is not generic enough for my cases, so my take is that it should be part of a private while listing it in .gitignore.

**Pull request closed without merging**